model/registry/process.yaml

```diff
@@ -41,6 +41,25 @@ groups:
           The PID of the process's group leader. This is also the process group
           ID (PGID) of the process.
         examples: [23]
+      - id: process.executable.build_id.gnu
+        stability: experimental
+        type: string
+        brief: >
+          The GNU build ID as can be found in the `.note.gnu.build-id` ELF section.
+        examples: ['c89b11207f6479603b0d49bf291c092c2b719293']
+      - id: process.executable.build_id.go
+        stability: experimental
+        type: string
+        brief: >
+          The Go build ID as it can be retrieved by `go tool buildid <go executable>`.
+        examples: ['foh3mEXu7BLZjsN9pOwG/kATcXlYVCDEFouRMQed_/WwRFB1hPo9LBkekthSPG/x8hMC8emW2cCjXD0_1aY']
+      - id: process.executable.build_id.profiling
+        stability: experimental
+        type: string
+        brief: >
+          Profiling specific build ID for executables. See the OTel specification
```

Comment:
Linking the specification is not possible at this moment, as https://github.com/open-telemetry/opentelemetry-specification/pull/4197 is pending.